Makefile

```diff
@@ -153,6 +156,21 @@ push-image: $(CONFIG_DOCKER_TARGET) build-image
 	@echo "Pushing the $(IMAGE_NAME) docker image for $(LOCAL_ARCH)..."
 	@docker push $(IMAGE_REPO)/$(IMAGE_NAME)-$(LOCAL_ARCH):$(VERSION)
 
+# By default create bundle image with manifests directory, if need to build other vesions,
+# for example build 1.1.0 bundle image, run command: BUNDLE_IMAGE_VERSION=1.1.0 BUNDLE_MANIFESTS_PATH=1.1.0 make bundle-image
+build-bundle-image: ## Create operator bundle image
+	@echo "Bulding the operator bundle image"
+	- $(OPERATOR_SDK) bundle create $(IMAGE_REPO)/$(BUNDLE_IMAGE_NAME)-$(LOCAL_ARCH):$(BUNDLE_IMAGE_VERSION) \
+	--directory deploy/olm-catalog/operand-deployment-lifecycle-manager/$(BUNDLE_MANIFESTS_PATH) \
+	--package operand-deployment-lifecycle-manager-app \
+	--channels stable-v1,beta,dev \
+	--default-channel stable-v1 \
+	--overwrite
```

Comment:
When I try with `operator-sdk` v0.15.1 and v0.19.0, they report `Error: unknown flag: --overwrite`.
Which `operator-sdk` version should we use?

Reply:
@horis233 to use this, your client operator-sdk version should be >=0.16.0, I use version 0.18.2 on my mac